README.md

```diff
@@ -57,13 +57,15 @@ Include `mh-opsworks-recipes` in your node's `run_list`:
   require a manual chef recipe invocation.
 * Rebase against develop and submit a pull request off of develop.
 
-## Releasing new chef recipes
+## QA'ing and releasing new chef recipes
 
 * Update CHANGELOG.md to collapse all "To be released" changes under a new
   version, following [semver](http://semver.org).
-* Rebase develop onto master.
-* Merge develop into master.
-* Tag the release on master.
+* Rebase develop onto master (just to be sure).
+* Cut an RC tag and link that into your staging cluster.
+* QA. When QA passes,
+* Merge the RC tag into master (which is hopefully the same as develop),
+* Tag the release on master based on the RC tag
```

Comment:
The  "(which is hopefully the same as develop)" has me confused. The RC tag allows more commits to go onto develop without affecting the "frozen" release candidiate that is being QA'd for production. Once the RC is tested and merged to master, the master branch should be the same as the RC (cut from develop), but not necessarily the same as develop. IMHO